message-me/app/views/sessions/new.html.erb

```diff
@@ -1 +1,38 @@
-<h1>this is new.html.erb for sessions</h1>
+<h1 class="ui header center aligned">Welcome to MessageMe - a complete Chat App</h1>
+<h4 class="ui center aligned medium icon header">
+  <i class="orange circular power off icon"></i>
+  Log in to continue
+</h4>
+
+<div class="ui placeholder segment">
+  <div class="ui two column very relaxed stackable grid">
+    <div class="column">
+      <div class="ui form">
```

Comment:
Why is this not built using rails form tag?

Reply:
Used Semantic UI's code 😅 would it be better to use rails form tag compared to this?

Reply:
definitely. 
Please tell me why?